packages/wrangler/src/index.tsx

```diff
@@ -1043,6 +1042,13 @@ export async function main(argv: string[]): Promise<void> {
               "Enter a secret value:",
               "password"
             );
+
+            if (args.local) {
+              return;
```

Comment:
How about some simple notification with `console.log...` that local mode don't go on writing secrets? 🤔

Reply:
It does log a warning, on line 1020